### DISCUSSION TITLE
Using fsh instead of sh.

### DISCUSSION THREAD

**Discussion by ajcasado** [EtchedPixels/FUZIX#1119] [General]
@EtchedPixels what’s the proper way to configure a platform to use fsh as the default shell instead of sh?

**Comment by EtchedPixels** [comment]
There isn't currently a nice way to do it or to set up things like multiple virtual terminals on systems that always have them. Until recently it used SDCC to build the userspace which was horribly slow and therefore everything was set up so I could build one file system and use it for all Z80.
Dragon does stuff with this so is worth a look.

**Reply by ajcasado** [reply]
Thanks for the advice, I'll take a look at it.
